gcc/config/riscv/vector.md

```diff
@@ -5933,6 +5933,58 @@
  [(set_attr "type" "vector")
   (set_attr "mode" "none")])
 
+;; Vector Mask Load/Store
+
+(define_expand "lm<mode>"
+  [(set (match_operand:VMASKMODES 0 "register_operand")
+	(unspec:VMASKMODES
+	  [(unspec:VMASKMODES
+	    [(mem:VMASKMODES (match_operand 1 "register_operand"))]
+	    UNSPEC_VLM)
+	   (reg:SI VL_REGNUM)]
+	 UNSPEC_USEVL))]
+  "TARGET_VECTOR"
+{
+})
+
+(define_insn "*lm<mode>"
```

Comment:
You don't need to separated `define_insn`/`define_expand` pattern if they have same RTL pattern, only `define_insn` is fine.

Reply:
They are not same completely, the expand need to translate address (register type) to memory type.

Reply:
Oh, I got it